### ISSUE TITLE
Augment new data to improve some classes detection

### ISSUE THREAD

**Issue by DataAugmentation** [ultralytics/yolov5#620]
## Additional context
As you see before augmenting data YOLO with pre-trained wight from the coco data set can detect all objects in Image
![before](https://user-images.githubusercontent.com/16304292/89209990-8f2b8b00-d5d4-11ea-8ec6-422ceca7ec5f.png)
but after adding new data and training the YOLO, It just can detect the people in a wheelchair.
![after](https://user-images.githubusercontent.com/16304292/89209946-7e7b1500-d5d4-11ea-921b-cf364be68fb5.png)
Also, below I provide more examples.
![before1](https://user-images.githubusercontent.com/16304292/89210390-445e4300-d5d5-11ea-8150-5cc3925ef7f9.png)
![after1](https://user-images.githubusercontent.com/16304292/89210616-ab7bf780-d5d5-11ea-9520-0403be559d7f.png)

**Comment by glenn-jocher** [comment]
@hbarzamini there's no such thing as only training one part of a neural network while not affecting the other parts. It's all interconnected.

**Comment by DataAugmentation** [comment]
> @hbarzamini there's no such thing as only training one part of a neural network while not affecting the other parts. It's all interconnected.

Then, when we retrain YOLO with new data and pre-trained weights, YOLO will forget old wights? So, every time we need to train YOLO from scratch with all the images?

**Comment by glenn-jocher** [comment]
Optimization always begins from initial conditions, and moves through the parameter space towards a lower loss region, by definition moving away from those initial conditions. If your training backpropagates a gradient that affects all parameters, then they will all change, regardless of where they started.
